ig/input/fsh/logical-adt-01.fsh

```diff
@@ -10,6 +10,21 @@ Description: "The expected input for the demographic data for the newborn screen
 * messageString 0..1 SU string "the full message as delimeted text"
 
 
+// history entries follow https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation#IGPublisherDocumentation-HistoryEntries
 
+Instance: adt01-initial-history
+InstanceOf: Provenance
+Title: "Initial creation of ADT01 message changelog"
+Usage: #definition
+* target[+] = Reference(Logical/adt-01-logical-model)
+* recorded = "2023-01-30T13:00:00.0000Z"
+* occurredDateTime = "2023-08-29"
+* reason = Code(http://terminology.hl7.org/CodeSystem/v3-ActReason, "METAMGT")
+* reason.text = "Initial Logical as per the C2S layout"
+* activity = Code(http://terminology.hl7.org/CodeSystem/v3-DataOperation, "CREATE")
```

Comment:
For the initial history entry this would be "CREATE", but subsequent will most likely be "UPDATE"

Reply:
Ok  that makes sense, thanks!